mailpoet/lib/Util/Notices/SenderDomainAuthenticationNotices.php

```diff
@@ -0,0 +1,167 @@
+    }
+
+    return Notice::displayWarning($noticeContent);
+  }
+
+  public function getNoticeContentForFreeMailUsers(int $contactCount): string {
+    if ($contactCount <= self::UPPER_LIMIT) {
+      // translators: %1$s is the domain of the user's default from address, %2$s is a rewritten version of their default from address, %3$s is HTML for an 'update sender' button, and %4$s is HTML for a Learn More button
+      return sprintf(__("<strong>Update your sender email address to a branded domain to continue sending your campaigns.</strong>
+<span>MailPoet can no longer send from email addresses on shared 3rd party domains like <strong>%1\$s</strong>. Please change your campaigns to send from an email address on your site's branded domain. Your existing scheduled and active emails will temporarily be sent from <strong>%2\$s</strong>.</span> <p>%3\$s &nbsp; %4\$s</p>", 'mailpoet'),
+        "@" . $this->getDefaultFromDomain(),
+        $this->authorizedSenderDomainController->getRewrittenEmailAddress($this->getDefaultFromAddress()),
+        $this->getUpdateSenderButton(),
+        $this->getLearnMoreAboutFreeMailButton()
+      );
+    }
+
+    // translators: %1$s is the domain of the user's default from address, %2$s is a rewritten version of their default from address, %3$s is HTML for an 'update sender' button, and %4$s is HTML for a Learn More button
+    return sprintf(__("<strong>Your newsletters and post notifications have been paused. Update your sender email address to a branded domain to continue sending your campaigns.</strong>
+<span>MailPoet can no longer send from email addresses on shared 3rd party domains like <strong>%1\$s</strong>. Please change your campaigns to send from an email address on your site's branded domain. Your marketing automations and transactional emails will temporarily be sent from <strong>%2\$s</strong>.</span> <p>%3\$s &nbsp; %4\$s</p>", 'mailpoet'),
+      "@" . $this->getDefaultFromDomain(),
+      $this->authorizedSenderDomainController->getRewrittenEmailAddress($this->getDefaultFromAddress()),
+      $this->getUpdateSenderButton(),
+      $this->getLearnMoreAboutFreeMailButton()
+    );
+  }
+
+  public function getNoticeContentForBrandedDomainUsers(bool $isPartiallyVerified, int $contactCount): string {
+    if ($isPartiallyVerified || $contactCount <= self::LOWER_LIMIT) {
+      // translators: %1$s is HTML for an 'authenticate domain' button, %2$s is HTML for a Learn More button
+      return sprintf(__("<strong>Authenticate your sender domain to improve email delivery rates.</strong>
+<span>Major mailbox providers require you to authenticate your sender domain to confirm you sent the emails, and may place unauthenticated emails in the \"Spam\" folder. Please authenticate your sender domain to ensure your marketing campaigns are compliant and will reach your contacts.</span><p>%1\$s &nbsp; %2\$s</p>", 'mailpoet'),
+        $this->getAuthenticateDomainButton(),
```

Comment:
From the Figma file, for partially verified domains, the button would always be secondary.
You might want to pass `$isPartiallyVerified` to this function to generate the class.

Reply:
Another good catch! This should also be resolved in 14bb974f1b387c898bd1eaa02cca1c1ae8ed1a53.